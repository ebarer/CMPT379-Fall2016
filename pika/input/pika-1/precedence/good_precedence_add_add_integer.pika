exec {

  print 82 + 23 + 48, _n_.
  print 23 + 57 + 49, _n_.
  print 66 + 90 + 52, _n_.
  print 59 + 1 + 58, _n_.
  print 63 + 23 + 35, _n_.
  print 26 + 34 + 18, _n_.
  print 73 + 6 + 53, _n_.
  print 58 + 14 + 34, _n_.
  print 46 + 79 + 15, _n_.
  print 25 + 27 + 62, _n_.
  print 59 + 96 + 41, _n_.
  print 46 + 45 + 50, _n_.

  print 10 + 85 + 3, _n_.
  print 55 + 17 + 14, _n_.
  print 72 + 76 + 27, _n_.
  print 18 + 52 + 51, _n_.
  print 77 + 45 + 92, _n_.
  print 51 + 21 + 71, _n_.
  print 5 + 76 + 72, _n_.
  print 69 + 67 + 58, _n_.
  print 29 + 15 + 29, _n_.
  print 15 + 13 + 13, _n_.
  print 31 + 35 + 73, _n_.
  print 93 + 3 + 84, _n_.

  print 45 + 55 + 29 + 64, _n_.
  print 93 + 97 + 39 + 40, _n_.
  print 67 + 49 + 48 + 43, _n_.
  print 20 + 9 + 41 + 55, _n_.
  print 99 + 75 + 3 + 30, _n_.
  print 96 + 56 + 48 + 11, _n_.
  print 30 + 66 + 69 + 28, _n_.
  print 56 + 49 + 99 + 20, _n_.
  print 48 + 45 + 96 + 30, _n_.
  print 76 + 67 + 70 + 53, _n_.
  print 38 + 22 + 44 + 53, _n_.
  print 7 + 58 + 69 + 12, _n_.

  print 63 + 26 + 75 + 37, _n_.
  print 66 + 4 + 96 + 74, _n_.
  print 12 + 63 + 67 + 23, _n_.
  print 41 + 80 + 52 + 94, _n_.
  print 42 + 48 + 35 + 53, _n_.
  print 1 + 58 + 29 + 87, _n_.
  print 48 + 66 + 79 + 44, _n_.
  print 99 + 64 + 13 + 27, _n_.
  print 19 + 4 + 6 + 69, _n_.
  print 14 + 96 + 21 + 97, _n_.
  print 61 + 5 + 46 + 24, _n_.
  print 81 + 32 + 36 + 54, _n_.

}

