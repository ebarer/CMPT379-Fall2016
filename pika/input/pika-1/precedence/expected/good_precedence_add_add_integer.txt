153
129
208
118
121
78
132
106
140
114
196
141
98
86
175
121
214
143
153
194
73
41
139
180
193
269
207
125
207
211
193
224
219
266
157
146
201
240
165
267
178
175
237
203
98
228
136
203
